.github/workflows/main.yml

```diff
@@ -320,7 +58,6 @@ jobs:
         with:
           context: images/plbase
           platforms: linux/amd64
```

Comment:
For lack of a better place to put this comment: we could in theory reuse the Python script to do these image builds, which would let us eliminate `scripts/check_path_modified.sh` entirely.

Reply:
I should also call out that I'm actually building the `plbase` image (and friends) multiple times: once here in order to run tests, and again to actually build/push the multiplatform image. We already actually do this (see `main.yml` and `images.yml`). Copy pasting my reasoning for why this is fine from a DM to Pete:

> This is generally fine on master as something can't get on master until it passes tests. I think this is also fine on PRs since we never push a latest tag there, and it can actually be helpful to rapidly make an image available for deployments.